.gitignore

```diff
@@ -3,7 +3,7 @@ dataSources/*_RAW.csv
 dataSources/*_RAW.xlsx
 scripts/.vscode/
 src/.snakemake/
-scripts/dag*
+src/dag*
 results/
 scripts/.ipynb_checkpoints/
 dataSources/UpdateDemand/

```

Comment:
There is a file called `scripts/__pycache__/functions.cpython-39.pyc` in this pull request. Can we add this to the GitIgnore, or is it needed?